### ISSUE TITLE
RTL Support: it works only on initialization

### ISSUE THREAD

**Issue by deounix** [mantinedev/mantine#633]
### What package has an issue

@mantine/core

### Describe the bug

There's a bug in `MantineProvider`  the RTL support is only apply one time (on initialization only). You can't remove OR add the RTL support after the init

Example:

```
const isRTL = true;
<MantineProvider
    children={children}
    emotionOptions={{
    key: "mantine",
    ...(isRTL && { stylisPlugins: [rtlPlugin] }),
    }}
    withNormalizeCSS
    withGlobalStyles
/>
```

This will work 100%

but after that if u want to change (isRTL) programmatically to false the RTL support wont removed and vise versa.

Note am using :

```
gatsby v4.4.0
@emotion/cache v11.7.1
```

### In which browser did the problem occur

Chrome

**Comment by rtivital** [comment]
Thanks for reporting, we will be working on all kind of rtl issues in one of the next 3.x releases

**Comment by rtivital** [comment]
Fixed in 3.6.0, see rtl guide – https://mantine.dev/theming/rtl/
